client/extensions/woocommerce/state/sites/payment-methods/test/actions.js

```diff
@@ -7,37 +7,68 @@ import { spy } from 'sinon';
 /**
  * Internal dependencies
  */
-import { fetchPaymentMethods } from '../actions';
+import { fetchPaymentMethods, savePaymentMethod } from '../actions';
 import { LOADING } from 'woocommerce/state/constants';
 import useNock from 'test/helpers/use-nock';
 import { useSandbox } from 'test/helpers/use-sinon';
 import {
 	WOOCOMMERCE_PAYMENT_METHODS_REQUEST,
 	WOOCOMMERCE_PAYMENT_METHODS_REQUEST_SUCCESS,
+	WOOCOMMERCE_PAYMENT_METHOD_UPDATE,
+	WOOCOMMERCE_PAYMENT_METHOD_UPDATE_SUCCESS,
 } from 'woocommerce/state/action-types';
 
 describe( 'actions', () => {
+	useSandbox();
+	useNock( ( nock ) => {
+		nock( 'https://public-api.wordpress.com:443' )
+			.persist()
+			.get( '/rest/v1.1/jetpack-blogs/123/rest-api/' )
+			.query( { path: '/wc/v3/payment_gateways&_method=get' } )
```

Comment:
I cannot since the change has not been made to request. If I change request.js then I have to fix all of the tests including the ones outside the scope of this PR.  I'd be happy to do modify this PR once that gets merged.